src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdaterTest.java

```diff
@@ -330,4 +117,60 @@ public void should_not_update_case_data_without_any_documents() {
         assertThat(exception).hasMessage("Missing scanned documents in exception record");
         verifyNoInteractions(addressExtractor);
     }
+
+    private void assertCaseData(SampleCase actualCaseData, SampleCase expectedCaseData) {
+        assertThat(actualCaseData)
+            .isEqualToComparingOnlyGivenFields(expectedCaseData,
+                "legacyId",
+                "firstName",
+                "lastName",
+                "dateOfBirth",
+                "bulkScanCaseReference"
+            );
+    }
+
+    private void assertScannedDocuments(
+        List<Item<ScannedDocument>> actualScannedDocuments,
+        List<Item<ScannedDocument>> caseScannedDocuments,
+        List<InputScannedDoc> exceptionRecordScannedDocuments
```

Comment:
Is this comparing actual to both `caseScannedDocuments` and `exceptionRecordScannedDocuments`.
This is very confusing.
Could there be 2 separate assertion methods instead?

Reply:
Done